src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarMain.test.tsx

```diff
@@ -14,23 +15,40 @@ jest.mock('./BrowserNavBarRegionSwitcher', () => () => (
   <div>BrowserNavBarRegionSwitcher</div>
 ));
 
+const props: BrowserNavBarMainProps = {
+  breakpointWidth: BreakpointWidth.TABLET
+};
+
 describe('BrowserNavBarMain', () => {
-  it('renders chromosome visualization by default', () => {
-    const wrapper = mount(<BrowserNavBarMain />);
+  let wrapper: any;
+
+  beforeEach(() => {
+    wrapper = mount(<BrowserNavBarMain {...props} />);
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('renders chromosome visualization by default for laptops or bigger screens', () => {
```

Comment:
then there should also be a test that checks that chromosome navigator does not get rendered for smaller screens

Reply:
I added that test

Reply:
✅